wisp-task/src/main/kotlin/wisp/task/RepeatedTaskMetrics.kt

```diff
@@ -14,24 +14,36 @@ import io.micrometer.core.instrument.MeterRegistry
  */
 class RepeatedTaskMetrics(meterRegistry: MeterRegistry) {
 
-    internal var taskDuration: DistributionSummary =
+    internal fun taskDuration(
+        meterRegistry: MeterRegistry,
+        name: String,
+        result: String,
+    ): DistributionSummary =
         DistributionSummary.builder("repeated.task")
             .description("count and duration in ms of periodic tasks")
-            .tag("name", "result")
+            .tag("name", name)
+            .tag("result", result)
             .percentilePrecision(4)
             .publishPercentiles(0.5, 0.75, 0.95, 0.99, 0.999)
             .publishPercentileHistogram()
             .register(meterRegistry)
 
-    internal var successCount: Counter = Counter.builder("repeated.task.success")
+    internal val successCount: Counter = Counter.builder("repeated.task.success")
```

Comment:
Did you mean to use the companion const vals here?

Reply:
Indeed.